.travis.yml

```diff
@@ -43,8 +50,28 @@ matrix:
     - rvm: 1.8.7
       gemfile: gemfiles/mime_types_2.3.gemfile
     - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.4.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.5.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.6.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.6_columnar.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.99.gemfile
+    - rvm: 1.8.7
       gemfile: gemfiles/mime_types_2.latest.gemfile
     - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_3.0.gemfile
+    - rvm: 1.9.2
+      gemfile: gemfiles/mime_types_3.0.gemfile
+    - rvm: 1.9.3
+      gemfile: gemfiles/mime_types_3.0.gemfile
```

Comment:
Probably. I haven’t yet had a chance to do anything with this—next week is a partial hack week for us, so I think I’ll be able to finish some of this stuff then.